packages/wdio-browserstack-service/src/service.js

```diff
@@ -80,7 +87,9 @@ export default class BrowserstackService {
 
     _getBody() {
         return {
-            status: this.failures === 0 ? 'completed' : 'error'
+            status: this.failures === 0 ? 'completed' : 'error',
+            name: this.name,
```

Comment:
`this.name` is defined no where

Reply:
I fixed the specific issue and fixed the tests by your review